views/demo.pug

```diff
@@ -4,13 +4,13 @@ block content
 	.container-fluid
 		.row
 			.col-sm-3.col-lg-2
-				nav.side-navbar.navbar-default.navbar-fixed-side
+				.side-navbar
 					ul
-						li
-							a(href="#") Link-1
-						li
-							a(href="#") Link-2
-						li
-							a(href="#") Link-3
+						a(href="#")
+							li Link-1
+						a(href="#")
```

Comment:
This is now backwards why are you wrapping li with a tags? Also you still are missing the point that you have to give all of these classes in component design. It's illegal to style HTML tags.